src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+public class Deadline extends Task {
+    String deadlineTime;
```

Comment:
I'm concern that some might interpret this variable to be a `LocalTime` object base on the name. Perhaps consider changing to `deadlineTimeString`

Reply:
Thanks for the suggestion!